fboss/platform/configs/icetea/platform_manager.json

```diff
@@ -1845,7 +1845,45 @@
           "busName": "SMB_MUX@1",
           "address": "0x4c",
           "kernelDeviceName": "tmp432",
-          "pmUnitScopedName": "SMB_TH6_SENSOR_TMP432_1"
+          "pmUnitScopedName": "SMB_TH6_SENSOR_TMP432_1",
+          "initRegSettings": [
```

Comment:
Please add a "__comment__" key in the json with value describing the reasoning for this change